opensquirrel/decomposer/aba_decomposer.py

```diff
@@ -45,8 +58,10 @@ def get_decomposition_angles(self, alpha: float, axis: AxisLike) -> tuple[float,
 
         """
         axis = Axis(axis)
-        a_axis_value = axis[self.index_a]
-        b_axis_value = axis[self.index_b]
+        a_axis_value = axis[self.index_a] + 0.0
+        b_axis_value = axis[self.index_b] + 0.0
+        c_axis_value = axis[self._find_non_used_index([self.index_a, self.index_b])] + 0.0
```

Comment:
If you accept the change above than this needs to change as well:
```suggestion
        c_axis_value = axis[self._find_non_used_index()] + 0.0
```